doclist-plugin/src/main/java/com/okta/maven/doclist/DocListMojo.java

```diff
@@ -85,10 +86,12 @@ public void execute() throws MojoExecutionException {
             FileUtils.copyURLToFile(getClass().getResource("/images/" + imageName), imageDest);
 
             // figure out the current version
-            TreeMap<String, String> versionsMap = new TreeMap<>(getVersions().stream()
-                    .collect(Collectors.toMap(v -> v, v -> v)));
+            NavigableMap<String, String> versionsMap = new TreeMap<>(getVersions().stream()
+                    .collect(Collectors.toMap(v -> v, v -> v))).descendingMap();
 
-            String currentVersion = project.getVersion();
+            String currentVersion = versionsMap.isEmpty()
+                                                    ? project.getVersion()
+                                                    : versionsMap.firstEntry().getKey();
```

Comment:
Since we're already sorting the versions [here](https://github.com/okta/okta-java-parent/blob/9023b55a0231a34fcbe186d85c48905b9420ae01/doclist-plugin/src/main/java/com/okta/maven/doclist/DocListMojo.java#L154), would it make sense to change the order in that method? It appears the `getVersions()` method isn't used elsewhere.

Then, you won't need this `descendingMap()` call.

Reply:
Updated, the `decendingMap()` is still needed, but i was able to remove the `sort` call in `getVersions()`